example/service/s3/getObjectWithProgress/README.md

```diff
@@ -0,0 +1,13 @@
+# Example
+
+This is an example using the AWS SDK for Go to download an S3 object with a progress bar.
+
+# Usage
+
+The example uses the bucket name provided, one key for object, and output the progress to stdout.
+
+```prompt
+AWS_PROFILE=cool-bucket AWS_REGION=us-west-2 go run getObjectWithProgress.go s3://cool-bucket/cool_thing.zip
```

Comment:
could you update this usage example to include the `-tag example` Go build tag as well.

Reply:
Done! 👍